firefox_ui_tests/functional/awesomebar/manifest.ini

```diff
@@ -0,0 +1 @@
+[test_star_in_autocomplete.py]
```

Comment:
We decided yesterday to have those tests located under `functional/locationbar`. Can you please do that in a follow-up?